gobblin-modules/gobblin-kafka-common/src/test/java/org/apache/gobblin/metrics/kafka/LoggingPusherTest.java

```diff
@@ -40,6 +41,7 @@ public void testKafkaReporter() {
 
     TestAppender testAppender = new TestAppender();
     Logger logger = LogManager.getLogger(LoggingPusher.class.getName());
+    logger.setLevel(Level.INFO);
```

Comment:
is this needed?

Reply:
Yes. It looks like the default Log level for tests is WARN, and this may be dependent on the specific logger implementation. But I observed flakiness running this test locally, due to the log level sometimes being set to WARN by default.